.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'type: bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
Some of these sections are not generally applicable (e.g. issues not be related to a front-end application). What about something like this:

```suggestion
**Describe the bug**
 A clear and concise description of the issue.
 
 **Your environment**
 Context as relevant - e.g. PlaceOS version, browser details, driver version etc.

 **To Reproduce**
 Steps to reproduce the behavior (if known).

 **Expected behavior**
 A clear and concise description of what you expected to happen.

 **Screenshots**
 If applicable, add screenshots to help explain your problem.

 **Additional context**
 Add any other context about the problem here.
```